src/github/deployment.ts

```diff
@@ -68,7 +68,13 @@ export const processDeployment = async (
 		logger
 	);
 
-	const result: DeploymentsResult = await jiraClient.deployment.submit(jiraPayload);
+	const result: DeploymentsResult = await jiraClient.deployment.submit(jiraPayload, webhookPayload.repository.id);
+
+	if (result.rejectedDeployments?.length) {
```

Comment:
In previous PR I moved this log into the jira-client. Do you mind remove this one here?

Reply:
Looks like it's still in the JC but the merge kept this one around for some reason. Have deleted.